src/io/tests/test_hdf_manual_reorder.cpp

```diff
@@ -0,0 +1,210 @@
+
+//////////////////////////////////////////////////////////////////////////////////////
+// This file is distributed under the University of Illinois/NCSA Open Source License.
+// See LICENSE file in top directory for details.
+//
+// Copyright (c) 2018 Jeongnim Kim and QMCPACK developers.
+//
+// File developed by: Luke Shulenburger, lshulen@sandia.gov, Sandia National Laboratories
+//
+// File created by: Luke Shulenburger, lshulen@sandia.gov, Sandia National Laboratories
+//////////////////////////////////////////////////////////////////////////////////////
+
+
+#include "catch.hpp"
+#include "io/hdf_archive.h"
+#include <iostream>
+#include <vector>
+#include <OhmmsPETE/OhmmsVector.h>
+#include <OhmmsPETE/OhmmsMatrix.h>
+#include <OhmmsPETE/OhmmsArray.h>
+
+using std::cout;
+using std::endl;
+using std::vector;
```

Comment:
I think this is best not to use since the code contains its own Vector type.  Also it is not used uniformly below.

Reply:
Fair enough, can clean this up and just explicitly use std:: in the unit test if needed.

Reply:
I think its worth while to make the tests as easy to understand as we can.